Makefile

```diff
@@ -85,3 +89,9 @@ docker-clickhouse:
 		-p $(DB_CLICKHOUSE_PORT):9000/tcp \
 		-l goose_test \
 		clickhouse/clickhouse-server:23-alpine
+
+docker-turso:
+	docker run --rm -d \
+		-p $(DB_TURSO_PORT):8080 \
+		-l goose_test \
+		ghcr.io/tursodatabase/libsql-server:v0.22.10
```

Comment:
Non-blocking : I wish this could also work on arm-based machines

> main_test.go:123: unexpected error: no matching manifest for linux/arm64/v8 in the manifest list entries

Afaics the docker image only supports amd64, 

https://github.com/tursodatabase/libsql/pkgs/container/libsql-server/158573424?tag=latest

Reply:
Oh, I didn't know this. I will open an issue to track